src/common/dnd/drag.directive.ts

```diff
@@ -33,13 +33,13 @@ export class DragDirective implements OnInit, OnDestroy {
 
   start(e) {
     const transfer = e.dataTransfer;
-    if (this.canDrag(this.event()) === false) {
+    if (this.canDrag === false) {
```

Comment:
from head.view.js:
canDrag = `map.hasOwnProperty(e.key)` && map[e.key].canMove !== false`
It's just boolean value